dev/release/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# This Dockerfile allows to build the package-registry and packages together.
+# It is decoupled from the packages and the registry even though for now both are in the same repository.
+ARG GO_VERSION=1.14.2
+FROM golang:${GO_VERSION}
+
+RUN \
+    apt-get update \
+      && apt-get install -y --no-install-recommends \
+         zip rsync \
+      && rm -rf /var/lib/apt/lists/*
+
+RUN git clone https://github.com/elastic/package-storage.git
```

Comment:
@mtojek I created the package-storage repository and now pull it in. I took some of the existing packages, copied them there with an older version so they are shipped but will not have any effect.

I realised this has an interesting side effect: Assuming we define that all packages in the storage layer must be under `/packages`, this would work with 1 or n repositories. More important, using Github for this is only one option, it could be any storage layer.

Reply:
Are you sure that the order Docker commands is the optimal one? Maybe it's better to build a repository code first. Also, `git clone` would pick up always the latest version of packages, not sure if it isn't better to depend on particular revisions.

Reply:
I doubt the order is optimal. At the moment it is all about getting something working we can iterate on. I hope in the end the container we build, will have a name that lets us identify which package registry version and which version of the packages was built so we could also reproduce it outside a docker container.

Reply:
Got it, thanks!